lib/wda/webdriveragent.js

```diff
@@ -96,22 +96,70 @@ class WebDriverAgent {
     }
   }
 
+  /**
+   * Return boolean if WDA is running or not
+   * @return {boolean} True if WDA is running
+   * @throws {Error} If there was invalid response code or body
+   */
   async isRunning () {
+    return !!(await this.getStatus());
+  }
+
+  /**
+   * Return current running WDA's bundle id
+   * @return {?string} bundle id
+   * @throws {Error} If there was invalid response code or body
+   */
+  async getRunningBundleId () {
+    const status = await this.getStatus();
+    if (util.hasValue(status)) {
+      try {
+        return status.build.productBundleIdentifier;
```

Comment:
Before https://github.com/appium/WebDriverAgent/pull/95 , we can't get `productBundleIdentifier`. It raises an error.